.vscode/launch.json

```diff
@@ -0,0 +1,44 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "serve web app",
+            "type": "chrome",
+            "request": "launch",
+            "preLaunchTask": "npm: start",
```

Comment:
So you´re aware this references a task that can be found in the `.vscode/tasks.json` file so i might not work as it is now.